projects/plugins/jetpack/babel.config.js

```diff
@@ -20,7 +20,17 @@ const config = {
 	env: {
 		test: {
 			presets: [ [ '@babel/preset-env', { targets: { node: 'current' } } ] ],
-			plugins: [ '@babel/plugin-transform-runtime' ],
+			plugins: [
+				'@babel/plugin-transform-runtime',
+				[
+					'module-resolver',
+					{
+						alias: {
+							'@automattic/jetpack-connection': '../../js-packages/connection',
+						},
+					},
+				],
```

Comment:
Can we avoid this mess with dependencies in package.json? Either with a dep like `link:../../js-packages/connection` or using [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) and a versioned dep?

Reply:
Good idea, I replaced the resolver with a Yarn link.